assignment/tool/src/webapp/vm/assignment/chef_assignments_list_assignments.vm

```diff
@@ -81,11 +76,35 @@
 			</div>
 		</div>
 		#end
-		<div class="sakai-table-pagerContainer">
-			#pagerPanel("pager1")
+		<div class="sakai-table-filterContainer">
```

Comment:
I think this should be inside 	#if ($hasAssignments) ?

Reply:
The filter selector is outside, because `$hasAssignments` is false if we don't have assignments for the current filter. Say, there are no drafts and you select that filter it will display the message that there are no assignments here, but the filter selector should still display to change the selection.

Reply:
Fair enough, thanks